pandas/io/formats/style_render.py

```diff
@@ -450,6 +461,105 @@ def _generate_col_header_row(self, iter: tuple, max_cols: int, col_lengths: dict
 
         return index_blanks + column_name + column_headers
 
+    def _generate_descriptor_row(self, iter: tuple, max_cols: int):
+        """
+        Generate the row containing calculated descriptor values for columns:
+
+         +----------------------------+---------------+---------------------------+
+         |  index_blanks ...          | descriptor_i  |  value_i by col           |
+         +----------------------------+---------------+---------------------------+
+
+        Parameters
+        ----------
+        iter : tuple
+            Looping variables from outer scope
+        max_cols : int
+            Permissible number of columns
+
+        Returns
+        -------
+        list of elements
+        """
+
+        r, descriptor = iter
+
+        # number of index blanks is governed by number of hidden index levels
+        index_blanks = [
+            _element("th", self.css["blank"], self.css["blank_value"], True)
+        ] * (self.index.nlevels - sum(self.hide_index_) - 1)
+
+        if isinstance(descriptor, str):
+            name: str | None = descriptor
```

Comment:
What happens if the method name doesn't reduce? e.g. `describe`?

Reply:
i think in that case the rendered cell output is the str representation of the result. I.e you can render a dataframe to the cell, albeit it might not look pretty.

Reply:
docs do say func should reduce to scalar, however